mne_bids/mne_bids.py

```diff
@@ -158,6 +158,68 @@ def _events_tsv(events, raw, fname, trial_type, verbose):
     return fname
 
 
+def _participants_tsv(raw, group, fname, verbose):
+    """Create a participants.tsv file and save it.
+
+    This will append any new participant data to the current list if it
+    exists
+
+    Parameters
+    ----------
+    raw : instance of Raw
+        The data as MNE-Python Raw object.
+    group : str
+        Name of group participant belongs to.
+    fname : str
+        Filename to save the participants.tsv to.
+    verbose : bool
+        Set verbose output to true or false.
+
+    """
+    subject_info = raw.info['subject_info']
+    if subject_info is not None:
```

Comment:
I think that there should be an `else` for the case where there is no `subject_info` ... perhaps a warning, that no participants.tsv can be generated and will have to be created manually?

Reply:
Sure, could get it to print a warning like when it finds no frequency and defaults to 50Hz

Reply:
+1, but please be a bit more explicit with the warning, e.g.: `warn('No participant data provided. Not writing a "participants.tsv" file.')`